troubleshooting/debugging.md

```diff
@@ -32,6 +32,7 @@
     - [Searching old issues](#searching-old-issues)
     - [Downgrading or upgrading your version](#downgrading-or-upgrading-your-version)
     - [Network Connection Issues](#network-connection-issues)
+    - [CloudWatch Broken Connection Issues](#cloudwatch-broken-connection-issues)
```

Comment:
I think that this is the natural place for this. We may value currently over the other issues because some customers are reaching out to ask about it.

However, it may be that they are reaching out and asking about it and not the other items which are listed before it, because these items are already in our debugging guide (and more important)

Reply:
Updated it to a section closer to the top of the debugging guide.